### PR TITLE
make save button activate after more actions

### PR DESCRIPTION
**Pull request by TomWoodward**
make edits referenced in http://redmine.oerpub.org/issues/275 activate the save button using fix mentioned in http://redmine.oerpub.org/issues/248

### CODE REVIEW

File: src/plugins/common/list/lib/list-plugin.js [hunk 1]
```diff
@@ -487,6 +487,7 @@ define([
 				}
 			}
 
+            Aloha.activeEditable.smartContentChange({type: 'block-change'})
```
Comment (philschatz) [inline src/plugins/common/list/lib/list-plugin.js]: Does the latest upstream version of Aloha already address these issues? I'm worried these small changes in many of the common plugins will make it more difficult to pull upstream changes

**Comment by kathi-fletcher** [comment]
I tested on testing.oerpub.org and made comments here http://redmine.oerpub.org/issues/275#change-1009 

The timing of certain things is weird and image and italics don't seem to be working.

File: src/plugins/common/list/lib/list-plugin.js (hunk 1 above)
Comment (TomWoodward) [inline src/plugins/common/list/lib/list-plugin.js]: I'm not super familiar with which plugins are ours and which are external dependancies - i definitely dont want to make any changes plugins that we dont maintain - if someone could make a note of which files i edited which i shouldn't have i can try to find another workaround

Comment (philschatz) [inline src/plugins/common/list/lib/list-plugin.js]: One reason we forked the Aloha-Editor project was to contribute back (fixing XHTML serialization to work with Math would be one example).

By far the majority of oerpub/cnx code is in `plugins/oer` and `plugins/contrib`.

**Comment by TomWoodward** [comment]
as much as i think and event based architecture would be more "javascritpy" it seems like the changed event is extremely unreliable, especially in some of the plugins notably the list one. so i elected to just poll the `isModified()` function instead. `isModified()` is the criteria for whether the save is actually handled when the button is clicked anyway, so it makes sense that they be lined up.

**Comment by philschatz** [comment]
It seems the `smartContentChange()` isn't quite smart enough yet.

**Pull request merged**